keyboards/helix/rev2/keymaps/froggy_106/keymap.c

```diff
@@ -0,0 +1,901 @@
+      ),
+  [_NUM_106] = LAYOUT( \
+      _______,  _______, OSL(_FUNC), KC_HOME, KC_END,  _______,                          _______,  _______,  _______,  _______,  _______,  _______, \
+      _______,  JP_ASTR, KC_P7,      KC_P8,   KC_P9,   JP_MINS,                          _______,  _______,  _______,  _______,  _______,  _______, \
+      KC_DOT,   JP_SLSH, KC_P4,      KC_P5,   KC_P6,   JP_PLUS,                          _______,  _______,  _______,  _______,  _______,  _______, \
+      KC_NLCK,  KC_P0,   KC_P1,      KC_P2,   KC_P3,   LCTL(S(KC_F1)), _______, _______, _______,  _______,  _______,  _______,  _______,  _______, \
+      _______,  _______, KC_PDOT,    JP_COMM, _______, _______,        _______, _______, _______,  _______,  _______,  _______,  _______,  _______ \
+      ),
+  
+  /* Func
+   * ,-----------------------------------------.             ,-----------------------------------------.
+   * |RGBRST|  Hue |To101 |  RST |  Mac |  Win |             |      |      |      |      |      |      |
+   * |------+------+------+------+------+------|             |------+------+------+------+------+------|
+   * | RGB1 | VAL+ |  F7  |  F8  |  F9  |To106 |             |      |      |      |      |      |      |
+   * |------+------+------+------+------+------|             |------+------+------+------+------+------|
+   * | RGB2 | VAL- |  F4  |  F5  |  F6  | F12  |             |      |      |      |      |      |      |
+   * |------+------+------+------+------+------+------+------+------+------+------+------+------+------|
+   * | RGB3 |  F10 |  F1  |  F2  |  F3  | F11  |      |      |      |      |      |      |      |      |
+   * |------+------+------+------+------+------+------+------+------+------+------+------+------+------|
+   * |RGBOFF|      |      |      |      |      |      |      |      |      |      |      |      |      |
+   * `-------------------------------------------------------------------------------------------------'
+   */
+  [_FUNC] = LAYOUT( \
+      RGBRST,RGB_HUI, TO_101,  RESET,   MAC,     WIN,                         _______,  _______,  _______,  _______,  _______,  _______, \
+      RGB1,  RGB_VAI, KC_F7,   KC_F8,   KC_F9,   TO_106,                      _______,  _______,  _______,  _______,  _______,  _______, \
+      RGB2,  RGB_VAD, KC_F4,   KC_F5,   KC_F6,   KC_F12,                      _______,  _______,  _______,  _______,  _______,  _______, \
+      RGB3,  KC_F10,  KC_F1,   KC_F2,   KC_F3,   KC_F11,   _______,  _______, _______,  _______,  _______,  _______,  _______,  _______, \
+      RGBOFF,_______, _______, _______, _______, _______,  _______,  _______, _______,  _______,  _______,  _______,  _______,  _______ \
+      )
+};
+#else
+#error "undefined keymaps"
+#endif
+
+
+#ifdef AUDIO_ENABLE
+
+float tone_qwerty[][2]     = SONG(QWERTY_SOUND);
+float tone_dvorak[][2]     = SONG(DVORAK_SOUND);
+float tone_colemak[][2]    = SONG(COLEMAK_SOUND);
```

Comment:
These shouldn't be needed. 
```suggestion
```

Reply:
Actually, sound feature is not implemented this hardware, so remove sound functions.